sysutils/node_exporter/src/opnsense/service/conf/actions.d/actions_node_exporter.conf

```diff
@@ -0,0 +1,23 @@
+[start]
+command: /usr/local/etc/rc.d/node_exporter start
+parameters:
+type: script
+message: Start Node Exporter Service
+
+[stop]
+command: /usr/local/etc/rc.d/node_exporter stop
+parameters:
+type: script
+message: Stop Node Exporter Service
+
+[restart]
+command: /usr/local/etc/rc.d/node_exporter restart
+parameters:
+type: script
+message: Restart Node Exporter Service
+
+[status]
+command: /usr/local/etc/rc.d/node_exporter status
```

Comment:
requires appending "; exit 0" to the command

Reply:
Updated